### ISSUE TITLE
App Icon in "Recent Builds" is incorrect [Xcode 14]

### ISSUE THREAD

**Issue by Sherlouk** [AvdLee/RocketSimApp#197]
**Describe the bug**

App icon in "Recent builds" is incorrect using the new Xcode beta.

**To Reproduce**
1. Using Xcode 14 and a "Single Size" app icon, build your iOS app.
2. Notice how the app icon in RocketSim is **not** the app's icon.

**Expected behavior**

I would expect the app icon in the "Recent builds" area to be that of the app (see the one on the simulator homepage).

**Screenshots**

![image](https://user-images.githubusercontent.com/15193942/176219980-7cbd4fc7-b42c-4661-a20c-eb69ebbe94c4.png)

**Desktop (please complete the following information):**

- Simulator: iPhone 3 Pro (iOS 16)
- macOS: v12.4 (Monterey)
- Xcode: v14 beta 1

**Additional context**

I believe the most likely root cause here is that I'm using the new "Single Size" app icon option in the xcassets. This means instead of providing what feels like 100,000 app icon sizes - you just provide one (1024x1024).

Looking at the generated .app file, the two icons below exist:

```
AppIcon60x60@2x.png
AppIcon76x76@2x~ipad.png
```

but the Info.plist has the following in it.

```xml
	<key>CFBundleIcons</key>
	<dict>
		<key>CFBundlePrimaryIcon</key>
		<dict>
			<key>CFBundleIconFiles</key>
			<array>
				<string>AppIcon1024x1024</string>
			</array>
			<key>CFBundleIconName</key>
			<string>AppIcon</string>
		</dict>
	</dict>
```

So I suspect RocketSim is either looking for a specific image, or the one provided in the plist, and either way is unable to find the right one.

**Comment by AvdLee** [comment]
Aii, interesting!! Thanks for reporting in, I'll have a look

**Comment by AvdLee** [comment]
I was able to fix this by adding a fallback to search for any PNG with `AppIcon` in the name. Fixed in 8.4.0!
